iceoryx_examples/iceperf/roudi_main_static_config.cpp

```diff
@@ -25,7 +25,15 @@ int main(int argc, char* argv[])
     static constexpr uint32_t ONE_MEGABYTE = 1024U * 1024;
 
     iox::config::CmdLineParserConfigFileOption cmdLineParser;
-    cmdLineParser.parse(argc, argv);
+    auto cmdLineArgs = cmdLineParser.parse(argc, argv);
+    if (cmdLineArgs.has_error())
+    {
+        if (cmdLineArgs.get_error() == iox::config::CmdLineParserResult::UNKNOWN_OPTION_USED)
```

Comment:
how about using `if (cmdLineArgs.get_error() != iox::config::CmdLineParserResult::INFO_OUTPUT_ONLY)` then we are safe if we forget to adjust this.
The two if statements could also be combined in a single one with `&&`

Reply:
Good idea, adapted as proposed.